index.js

```diff
@@ -197,10 +197,11 @@ BabelClient.prototype.getAnnotations = function(token, querystringMap, callback)
  * @param {string} data.annotatedBy
  * @param {string} data.motiviatedBy
  * @param {string} data.annotatedAt
- *
- * @callback callback
+ * @param {object} options that control the request being made to babel.
+ * @param {boolean} options.xIngestSynchronously
+ * @param callback
  */
-BabelClient.prototype.createAnnotation = function(token, data, callback){
+BabelClient.prototype.createAnnotation = function(token, data, options, callback){
```

Comment:
This is a breaking change from the last version. Probably need to update the version history in the README to say so.

Also - is it worth doing something here so that it's not a breaking change? (i know its just us that's using it right now) - if ```options``` is a typeof function, then set the callback to options?